src/platform/platform-device-info.h

```diff
@@ -35,10 +35,10 @@ class platform_device_info : public device_info
 
     std::string get_address() const override
     {
-        for( auto & d : _group.uvc_devices )
-            return d.device_path;
-        for( auto & d : _group.usb_devices )
-            return d.id;
+        if( _group.uvc_devices.size() )
+            return _group.uvc_devices.front().device_path;
+        if( _group.usb_devices.size() )
```

Comment:
What was the Coverity complaint here? I see nothing wrong with this code that I wrote :)

Reply:
`CID 6053338 (#4 of 4): Structurally dead code (UNREACHABLE)
unreachable: Since the loop increment ++<anonymous>; is unreachable, the loop body will never execute more than once.`

I also think that your code is fine, but Tamir's change keeps the original logic and suppresses the warning.
If you have objections we can consider marking this issue as `ignore`. It will keep showing in the reports but will not count as a reason to fail the security SWRC

Reply:
What a waste of time...